README.rst

```diff
@@ -52,16 +52,17 @@ been imported via:
 
 .. code::
 
-    from stdio_mgr import stdio_mgr
+    >>> from stdio_mgr import stdio_mgr
 
 **Mock** ``stdout``\ **:**
 
 .. code::
 
+    >>> import os
     >>> with stdio_mgr() as (in_, out_, err_):
     ...     print('foobar')
     ...     out_cap = out_.getvalue()
```

Comment:
How about you move the `os.linesep`s up into the contexts, e.g. here, to where `out_cap` is retrieved from `out_`? That would make for cleaner REPL-prints.

Reply:
Done.
And created https://github.com/bskinn/stdio-mgr/issues/80